trinity/main.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import asyncio
+import atexit
+
+from evm.db.chain import BaseChainDB
+from evm.exceptions import CanonicalHeadNotFound
+from evm.chains.ropsten import (
+    ROPSTEN_GENESIS_HEADER,
+    ROPSTEN_NETWORK_ID,
+)
+from evm.chains.mainnet import MAINNET_VM_CONFIGURATION
+from evm.p2p import ecies
+from evm.p2p.lightchain import LightChain
+from evm.db.backends.level import LevelDB
+
+from trinity.utils.filesystem import (
+    ensure_path_exists,
+)
+from trinity.utils.logging import (
+    setup_trinity_logging,
+    setup_queue_logging,
+)
+from trinity.utils.mp import (
+    ctx,
+)
+from trinity.utils.xdg import (
+    get_data_dir,
+)
+
+
+DEFAULT_LOG_LEVEL = 'info'
+LOG_LEVEL_CHOICES = (
+    'debug',
+    'info',
+)
+
+
+parser = argparse.ArgumentParser(description='Trinity')
```

Comment:
So we landed on argparse? :-/

Ok, just had to get in one last *grumble*. :)

Reply:
No, but it's the easiest one to replace with the other tools and I want to see if we can get by with it.  So far it looks like it's going to be fine.

Reply:
Ok, fair enough. In the abstract, I do prefer using stdlib tools, so if it's not broken then :+1: .